src/processor_arm.cpp

```diff
@@ -1586,7 +1588,7 @@ static uint32_t sysimg_init_cb(const void *id)
     return match.best_idx;
 }
 
-static uint32_t pkgimg_init_cb(const void *id)
+static uint32_t pkgimg_init_cb(const void *id, jl_value_t** rejection_reason)
```

Comment:
You can mark all of these JL_REQUIRE_ROOTED_SLOT to make sure the gc-analyzer doesn't get unhappy
```suggestion
static uint32_t pkgimg_init_cb(const void *id, jl_value_t **rejection_reason JL_REQUIRE_ROOTED_SLOT)
```